src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestUpdater.cs

```diff
@@ -320,9 +320,14 @@ private async Task<MergePolicyCheckResult> CheckMergePolicyAsync(InProgressPullR
         {
             await remote.MergeDependencyPullRequestAsync(pr.Url, new MergePullRequestParameters());
         }
-        catch
+        catch (PullRequestNotMergeableException notMergeableException)
+        {
+            _logger.LogInformation("NOT Merged: PR '{url}' cannot be merged - {message}", pr.Url, notMergeableException.Message);
```

Comment:
Change the messages here please so it differs from the one below and we can distinguish those

Reply:
updated